File: common_java_lib/common/src/com/whaley/biutil/BiStartEndLog.java
package com.whaley.biutil;

/**
 * Created by wanglei on 2016/7/29.
 */
public class BiStartEndLog extends BiLog {
    protected String Uuid;
    protected String Status;
    public BiStartEndLog(String caseId, String uuid, String status) {
        super("start_end");
	PutHeader("status", status);
	PutHeader("actionId", caseId);
	PutHeader("uuid", uuid);
    }
}

File: common_native_lib/bistartendlog.cpp
/*
 * Yuchuan Sun
 * Version 1.0
 * Date 2016-8-5
 * BiLog common module that is used by native code
*/

#define LOG_TAG "BiStartEndLog"
#include "bistartendlog.h"
#include <utils/Log.h>

BiStartEndLog::BiStartEndLog(const char *caseId,
    const char *uuid, const char *status):
    BiLog("start_end")
{
    ALOGI("constructor");
    PutHeader("status", status);
    PutHeader("actionId", caseId);
    PutHeader("uuid", uuid);
}

BiStartEndLog::~BiStartEndLog()
{
    ALOGI("destructor");
}

